docs/contracts.rst

```diff
@@ -519,8 +519,7 @@ Ether (without data). Additionally, in order to receive Ether, the fallback func
 must be marked ``payable``. If no such function exists, the contract cannot receive
 Ether through regular transactions.
 
-In such a context, there is usually very little gas available to the function call (to be precise, 2300 gas),
-so it is important to make fallback functions as cheap as possible.
+In such a context, there is usually very little gas available to the function call (to be precise, 2300 gas), so it is important to make fallback functions as cheap as possible. Note that the gas required by a transaction (as opposed to an internal call) that invokes the fallback function is much higher, because each transaction charges an additional amount of 21000 gas or more for things like signature checking.
```

Comment:
I think this is a bit confusing, it seems like 21000 gas is charged so the contract can do signature checking (which is not the case).

However, that 21000 is not usable by the contract if I am not mistaken?

Reply:
It's not usable by the contract, no. I don't think it reads like what you said though. But ofcourse if you think it can be expressed in some more clear way let's change it.